atom/browser/native_window_mac.mm

```diff
@@ -819,7 +826,9 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
       zoom_to_page_width_(false),
       fullscreen_window_title_(false),
       attention_request_id_(0),
-      title_bar_style_(NORMAL) {
+      title_bar_style_(NORMAL),
+      simple_fullscreen_(false),
```

Comment:
Having both `simple_fullscreen_` and `is_simple_fullscreen_` is confusing, renaming `simple_fullscreen_` to something like `always_simple_fullscreen_` can improve code readability.

Reply:
Good call, I can see how the current nomenclature is ambiguous. I will update accordingly.